src/config_parser.cpp

```diff
@@ -1,92 +1,113 @@
 #include "stdafx.h"
 #include "config_parser.h"
 
+const static json s_cDefaultJson = {
+        {"serviceName", "JesusIncService"},
+        {"diskFreeSpace", 90U},
+        {"hasInternetConnection", true},
+        {"logPath", "../"},
+        {"logName", "log.log"},
+        {"cpuLoad", 40U}
+};
+const static std::string s_csConfigFileName = "../config/config.json";
+const static std::string s_csParamServiceName = "serviceName";
+const static std::string s_csParamLogPath = "logPath";
+const static std::string s_csParamLogName = "logName";
+
+//const static std::string s_csParamDiskFreeSpace = "diskFreeSpace";
+//const static std::string s_csParamCpuLoad = "cpuLoad";
+//const static std::string s_csParamInternetConnection = "hasInternetConnection";
+
 CConfigParser::CConfigParser()
 {
     m_UserJson = s_cDefaultJson;
-    ParseConfigFile();
-    if (m_bConfigExists && m_bConfigParseSuccsessful)
-    {
-        CompareConfigFiles(s_cDefaultJson, m_UserJson);
-        CheckValuesValidity();
-    }
-    InitializeMembers();
-    LogServiceData();
 }
 
-std::string CConfigParser::GetLogData()
-{
-    return m_Oss.str();
-}
+//int CConfigParser::GetDiskFreeSpace()
+//{
+//    return m_iDiskFreeSpace;
+//}
 
-int CConfigParser::GetDiskFreeSpace()
-{
-    return m_iDiskFreeSpace;
-}
+//int CConfigParser::GetCpuLoad()
+//{
+//    return m_iCpuLoad;
+//}
 
-int CConfigParser::GetCpuLoad()
-{
-    return m_iCpuLoad;
-}
+//bool CConfigParser::GetHasInternetConnection()
+//{
+//    return m_bHasInternetConnection;
+//}
 
-bool CConfigParser::GetHasInternetConnection()
+std::string CConfigParser::GetServiceName()
 {
-    return m_bHasInternetConnection;
+    return m_sServiceName;
 }
 
-std::string CConfigParser::GetServiceName()
+std::string CConfigParser::GetLogPath()
 {
-    return m_sServiceName;
+    return m_sLogPath;
 }
 
-void CConfigParser::LogServiceData()
+std::string CConfigParser::GetLogName()
 {
-    m_Oss << " >> Service parameters:\n";
-    m_Oss << "\tService name: " << GetServiceName() << "\n";
-    m_Oss << "\tFree disk space: " << GetDiskFreeSpace() << "\n";
-    m_Oss << "\tCpu loaded: " << GetCpuLoad() << "\n";
-    m_Oss << "\tInternet connection available: " << GetHasInternetConnection() << "\n";
+    return m_sLogName;
 }
 
-void CConfigParser::ParseConfigFile()
+void CConfigParser::ParseConfigFile(std::ostringstream& oss)
 {
-    m_bConfigExists = std::filesystem::exists(s_csConfigFileName);
+    bool m_bConfigExists = std::filesystem::exists(s_csConfigFileName);
+    bool m_bConfigParseSuccsessful = false;
```

Comment:
Does 'm_' prefix stand for member variable in your code style? Because looks like these are local variables.